maintenance/removeDuplicateEntities.php

```diff
@@ -69,6 +71,16 @@ public function execute() {
 
 		$duplicateEntityRecords = $duplicateEntitiesDisposer->findDuplicates();
 		$duplicateEntitiesDisposer->verifyAndDispose( $duplicateEntityRecords );
+		
+		if ( $result && $this->hasOption( 'report-runtime' ) ) {
```

Comment:
This fails because `$result` is not defined, so checking for `if ( $this->hasOption( 'report-runtime' ) )` should be sufficient.

```
There was 1 error:
1) SMW\Tests\Integration\Maintenance\RemoveDuplicateEntitiesTest::testRun
Undefined variable: result
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/maintenance/removeDuplicateEntities.php:75
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/Utils/Runners/MaintenanceRunner.php:103
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/Integration/Maintenance/RemoveDuplicateEntitiesTest.php:41
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/DatabaseTestCase.php:138
/home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:94
```

Reply:
Thanks a lot for the hint. Done!